includes/polling/ports.inc.php

```diff
@@ -212,6 +216,7 @@
         $walk_base = $total_port_count - $polled_port_count < 5 || $polled_port_count / $total_port_count > 0.9 ;
 
         if ($walk_base) {
+            echo "Not enough ports for selected port polling, walking base OIDs instead\n";
```

Comment:
This could be improved.  We could split out the $walk_base boolean and accurately describe why the walk is being performed.

Reply:
Seems pointless as I really do want to remove this code.

Reply:
Let's look at that in a different PR.  This PR looks good to me, just the inline of that array + remove the implode there.

Reply:
I've sorted the array out but left the text as is. Seems a bit ott to set variables for both those outcomes just to be able to say which one caused the selected port polling to fail.